src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const dataSalary = Array.from(document.querySelectorAll('Li'));
+
+function sortList(list) {
+  const sortElement = list.sort((a, b) => {
+    const solaryA = Number(a.getAttribute('data-salary').replace(/[$,]/g, ''));
+    const solaryB = Number(b.getAttribute('data-salary').replace(/[$,]/g, ''));
```

Comment:
There seems to be a typo in the variable name 'solaryB'. It should likely be 'salaryB' to accurately reflect its purpose.